### PR TITLE
Add_link_broadband_tariffs_internet_access

### PR DESCRIPTION
**Pull request by amylaurence1**
### Context

DCMS/DoE request to signpost users to Ofcom's webpage that outlines the different social tariff offers available.

### Changes proposed in this pull request

Added a link to Ofcom's webpage.

### CODE REVIEW

**Review by Lloyd-K — APPROVED** [review]

File: app/views/pages/internet_access.html.erb [hunk 1]
```diff
@@ -15,6 +15,10 @@
       How to request internet access for disadvantaged pupils in <span class="app-no-wrap">years 3 to 13</span>, students in further education, and care leavers.
     </p>
 
+<div class="govuk-inset-text">
+      Young people and families on low incomes or certain benefits may be eligible for cheaper broadband tariffs. <%= govuk_link_to 'Find out who is eligible and which providers offer targeted tariffs', 'https://www.ofcom.org.uk/about-ofcom/latest/features-and-news/support-for-customers-during-pandemic' %>
```
Comment (ralph-hawkins) [inline app/views/pages/internet_access.html.erb:19]: Should "Find out who is eligible and which providers offer targeted tariffs" have a full stop (outside the link?) sorry I know this is my fault.

Comment (Lloyd-K) [reply app/views/pages/internet_access.html.erb:19]: Yes, thanks for spotting!

Comment (amylaurence1) [reply app/views/pages/internet_access.html.erb:19]: You're right @ralph-hawkins - don't apologise!

**Review by Lloyd-K — APPROVED** [review]

**Review by Lloyd-K — APPROVED** [review]

**Review by AMcContent — COMMENTED** [review]
All good

**Comment by AMcContent** [comment]
Reviewed

**Pull request merged**